src/query/graphite/native/aggregation_functions.go

```diff
@@ -135,6 +135,29 @@ func divideSeries(ctx *common.Context, dividendSeriesList, divisorSeriesList sin
 	return r, nil
 }
 
+// aggregate takes a list of series and returns a new series containing the
+// value aggregated across the series at each datapoint using the specified function.
+func aggregate(ctx *common.Context, series singlePathSpec, fname string) (ts.SeriesList, error) {
+	switch fname {
+	case Empty, Sum, SumSeries, Total:
+		return sumSeries(ctx, multiplePathSpecs(series))
+	case Min, MinSeries:
+		return minSeries(ctx, multiplePathSpecs(series))
+	case Max, MaxSeries:
+		return maxSeries(ctx, multiplePathSpecs(series))
+	case Avg, Average, AverageSeries:
+		return averageSeries(ctx,multiplePathSpecs(series))
```

Comment:
nit: Need a space between args here.

Reply:
Though go format tool should have fixed that. but yes, let me fix.